packages/joint-core/docs/src/joint/api/dia/CellView/prototype/findBySelector.html

```diff
@@ -1,12 +1,9 @@
-<pre class="docs-method-signature"><code>cellView.findBySelector([selector, root, selectors])</code></pre>
-<p>Return an array of subelements of this CellView which are identified by <code>selector</code>. If no such subelements are found, return an empty array.</p>
+<pre class="docs-method-signature"><code>cellView.findBySelector(selector)</code></pre>
 
-<p>This method mimics jQuery element search methods in that it always returns an array. If you are only expecting to find a single subelement (most common scenario), you can get it by accessing the first element in the returned array – e.g. <code>this.findBySelector('body')[0]</code>.</p>
+<p>The method returns an array of DOM nodes matching the <code>selector</code>.</p>
 
-<p>If you do not specify a <code>selector</code>, the <code>root</code> of the CellView is returned as the only element of the return array. For backwards compatibility, the same applies also if you provide the special value <code>selector = '.'</code>. By default <code>root = this.el</code> (the wrapping <code>&lt;g&gt;</code> SVGElement of the CellView), but if you provide a different <code>root</code> (see below), then that value is returned instead.</p>
+<p>The method searches for matching sub-elements among the descendants of this <code>this.el</code>. If no such sub-elements are found, an empty array is returned.</p>
```

Comment:
`this <code>this.el</code>.` - Not sure if `this` before the code block is there by accident.